frontend/index.html

```diff
@@ -20,6 +20,15 @@ <h3 id="door_status">Status: Unknown</h3>
 			<button id="btn_lock">Lock</button>
 			<button id="btn_unlock">Unlock</button>
 
+			<h2>Card Scanner</h2>
+			<label>Add card id (separated by comma, lowercase):</label><br><br>
+			<input id="add_card" placeholder="Enter Card Id"><br><br>
```

Comment:
Ideally we would like the user to be able to scan their card id instead of inputting it manually (how many people want to input a card id).
User hits add new card -> scanner becomes active  and user scans the new card-> display id on web page -> user hits confirm to add it.

(optional due to time)

Reply:
Yeah, that'll be better. I just made those changes.